install/kubernetes/helm/istio/charts/sidecar-injector/templates/configmap.yaml

```diff
@@ -18,33 +18,33 @@ data:
         image: {{ .Values.global.hub }}/proxy_init:{{ .Values.global.tag }}
         args:
         - "-p"
-        - {{ "{{ .MeshConfig.ProxyListenPort }}" }}
+        - {{ "[[ .MeshConfig.ProxyListenPort ]]" }}
```

Comment:
Do we still need the quotes with this delimiter change?

Reply:
I think we'd better keep them to avoid unnecessary spaces.